From 823b6754a4d7655480b6e8576a9d0037f842d653 Mon Sep 17 00:00:00 2001
From: Jan Luebbe <jlu@pengutronix.de>
Date: Thu, 25 Aug 2022 12:19:16 +0200
Subject: [PATCH] tools:gensiot: Fix build with musl

According to POSIX getpid() is available in unistd.h, not sys/unistd.h.

Upstream-Status: Submitted [https://github.com/cminyard/gensio/pull/47]
Signed-off-by: Jan Luebbe <jlu@pengutronix.de>
---
 tools/gensiotool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/gensiotool.c b/tools/gensiotool.c
index cac531bb4b74..ab0bb9583f9f 100644
--- a/tools/gensiotool.c
+++ b/tools/gensiotool.c
@@ -44,7 +44,7 @@
 #include <signal.h>
 #include <errno.h>
 #include <sys/types.h>
-#include <sys/unistd.h>
+#include <unistd.h>
 #include <syslog.h>
 #endif
 
-- 
2.30.2

